src/react-apps/ux-editor/src/containers/Container.tsx

```diff
@@ -0,0 +1,197 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import ApiActionDispatchers from '../actions/apiActions/apiActionDispatcher';
+import ConditionalRenderingActionDispatcher from '../actions/conditionalRenderingActions/conditionalRenderingActionDispatcher';
+import FormDesignerActionDispatchers from '../actions/formDesignerActions/formDesignerActionDispatcher';
+import FormFillerActionDispatchers from '../actions/formFillerActions/formFillerActionDispatcher';
+import RuleConnectionActionDispatchers from '../actions/ruleConnectionActions/ruleConnectionActionDispatcher';
+import { FormComponentWrapper } from '../components/FormComponent';
+import { IFormLayoutState } from '../reducers/formDesignerReducer/formLayoutReducer';
+
+export interface IProvidedContainerProps {
+  id: string;
+  baseContainer?: boolean;
+}
+
+export interface IContainerProps extends IProvidedContainerProps {
+  dataModelGroup?: string;
+  itemOrder: any;
+  components: any;
+  containers: any;
+  repeating: boolean;
+  designMode: boolean;
+  formData: any;
+  index?: number;
+}
+
+export class ContainerComponent extends React.Component<IContainerProps> {
+  
+  public handleContainerDelete = (e: any) => {
+    FormDesignerActionDispatchers.deleteFormContainer(this.props.id);
+    e.stopPropagation();
+  }
+
+  public handleComponentDataUpdate = (
+    id: string,
+    dataModelElement: IDataModelFieldElement,
+    callbackValue: any,
+  ): void => {
+    if ((this.props.index || this.props.index > -1) && this.props.dataModelGroup && this.props.repeating) {
+      const dataBindingName = dataModelElement.DataBindingName.replace(this.props.dataModelGroup,
+        this.props.dataModelGroup + `[${this.props.index}]`);
+      FormFillerActionDispatchers.updateFormData(
+        id,
+        callbackValue,
+        dataModelElement,
+        dataBindingName,
+        false,
```

Comment:
We don't support validating repeating groups yet? Is that why validate=false?

Reply:
We do, this was old code, is now removed.